docs/organizations-and-projects.mdx

```diff
@@ -0,0 +1,15 @@
+# Organizations and Projects
+
+## Organizations
+Organizations are the top-level umbrella for all the projects in a company. A user can create multiple organizations to separate projects under different companies or teams.
+An organization can have multiple users who have access to all projects under the organization. Users can invite other users to join and manage the organization. Check out the [Invite Users](/docs/invite-users) page for more information on inviting users to an organization.
+
+The name of the organization can typically be the name of the company or the department.
+
+Example: **My Company**.
+
+## Projects
+Projects are the workspaces for profile data within an organization, mainly to collect profiles that belong to a specific application or environment. Profiles within a project can be compared and analysed against each other as required.
+There can be multiple projects within an organization, and the users of an organization have access to all projects within the organization.
+
+Example: **Production Environment**.
```

Comment:
Similar to my comment on how to use organizations, we should make a recommendation on how to use projects. I think something like: "We recommend starting with creating one project and writing all data to a single project, that allows for the best visibility across the entire infrastructure."

Reply:
done, added this.